tests/testthat/test-eta_squared.R

```diff
@@ -615,6 +615,8 @@ if (require("testthat") && require("effectsize")) {
     expect_output(print(out), "Type II")
 
     skip_if_not_installed("car")
+    skip_if(getRversion() <= "3.6" &&
+              Sys.info()["sysname"] == "Darwin")
```

Comment:
you could also use `skip_on_os("mac")`?

Reply:
But it does work on newer version of R on Mac 🤷‍♂️

Reply:
Yes, I just meant to replace sys.info